ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -55,14 +56,22 @@ public class MySqlNamedBlobDbConfig {
    * A flag on whether to turn on DB Transition run
    */
   @Config(DB_TRANSITION)
-  @Default("false")
+  @Default("true")
```

Comment:
You really don't need to change the @Default. it is not used at all.

Reply:
Sounds good, removed.